README.md

```diff
@@ -90,6 +90,20 @@ Prawn::Labels.generate("names.pdf", names, :type => "QuarterSheet") do |pdf, nam
 end
 ```
 
+**Remember**: Prawn expects all [measurements](https://github.com/prawnpdf/prawn/blob/2f894ade8e6a97453a6a77fdf5c7400ca4dd7428/lib/prawn/measurements.rb) to be provided in Postscript points
```

Comment:
@pjaspers I'd rather we describe back to the manual conversion which is referenced here: https://github.com/jordanbyron/prawn-labels/blob/master/lib/prawn/types.yaml#L3